### DISCUSSION TITLE
Is possible to customize the OpenWebUI application database and Chroma victor database?

### DISCUSSION THREAD

**Discussion by SharpLu** [open-webui/open-webui#4288] [Q&A]
Hi Folks!

Hope you all is good!
I'm new for the OpenWebUI, just have a few questions if anyone can help with?

1.Can we customize the Chroma database (chroma.sqlite3) instead of the local we can have a remotely server to host the data?
2.Can we customize the the OpenWebUI application database (webui.db) to postgres or mysql?
3.If user uploaded documents will be shared globally for the openWebUI
4.how to manage the user uploaded files, is there have anyway to delete them, because i saw all users uploaded files will be written to the chroma database.

Thank you so much!

**Comment by TedLin1993** [comment]
1/2. You can set environment variable with `CHROMA_DATABASE` and `DATABASE_URL`
You can see document: https://docs.openwebui.com/getting-started/env-configuration
3/4. I think uploaded documents only save on our local database

**Reply by SharpLu** [reply]
Hi @TedLin1993  Thank you for your help!
Another following question
1.Do OpenWebUI support the pgvector now? can we config the pgvictor instead of the Chroma when deploy the OpenWebUI application? based on my search seems its not possible at this moment 

2.How about the user roles, can we setup multiple workspace and restricted by user group? does this supported by OpenWebUI now?

Thank you

**Reply by TedLin1993** [reply]
1. I think there is no support for pgvector right now.
2. Not right now, but I think there will be a better user management: https://docs.openwebui.com/roadmap
